pkg/gofr/http/errors_test.go

```diff
@@ -111,3 +111,19 @@ func Test_ErrorErrorPanicRecovery(t *testing.T) {
 
 	assert.Equal(t, http.StatusInternalServerError, err.StatusCode(), "TEST Failed.\n")
 }
+
+func Test_MultipleErrors(t *testing.T) {
+	err := MultipleErrors{Code: http.StatusNotFound, Errors: []error{ErrorInvalidRoute{}}}
+
+	assert.Equal(t, "route not registered", err.Error(), "TEST Failed.\n")
```

Comment:
```suggestion
assert.ErrorContains(t, err, "route not registered", , "TEST Failed.\n")
```

otherwise it would fail with a panic if err is nil

Reply:
err is defined on line 116, so it can't be nil.

Reply:
Oh you're not calling a method but testing the Error() I get it now, I'm sorry